### PR TITLE
Tinymce: Fix character count issue.

### PR DESCRIPTION
**Pull request by awais786**
https://openedx.atlassian.net/browse/ECOM-7693

### CODE REVIEW

**Comment by awais786** [comment]
**Without any character**

screen shot 2017-04-12 at 4 30 00 pm

**Comment by awais786** [comment]
**With characters**

screen shot 2017-04-12 at 4 31 20 pm

**Review by waheedahmed — APPROVED** [review]
looks good, just fix the build.

File: course_discovery/apps/publisher/utils.py [hunk 1]
```diff
@@ -108,3 +111,21 @@ def has_role_for_course(course, user):
         bool: True, if user has a role for course; otherwise, False.
     """
     return course.course_user_roles.filter(user=user).exists()
+
+
+def validate_text_count(max_length):
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/utils.py]: You can move this into `course_discovery/apps/publisher/validators.py`

**Pull request merged**